src/util/GCDMailbox.cc

```diff
@@ -122,22 +122,22 @@ namespace litecore { namespace actor {
             dispatch_async(_queue, wrappedBlock);
     }
 
-
     void GCDMailbox::afterEvent() {
-        _actor->afterEvent();
-        endBusy();
 #if ACTORS_TRACK_STATS
         ++_callCount;
         if (_eventCount > _maxEventCount) {
             _maxEventCount = _eventCount;
         }
 #endif
         --_eventCount;
+        
+        _actor->afterEvent();
```

Comment:
This still decrements `_eventCount` before `computeStatus` runs, so it would now have to compare it against 0 instead of 1.